Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -69,18 +88,16 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
         label.numberOfLines = 0
         label.adjustsFontForContentSizeCategory = true
         label.clipsToBounds = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
-                                                            size: UX.labelFontSize)
+        label.font = UX.baseFont
     }
 
     private lazy var subHeadlineLabel: UILabel = .build { label in
-        label.text = .Shopping.ReviewQualityCardSubHeadlineLabel
+        let text = String.Shopping.ReviewQualityCardSubHeadlineLabel
+        label.attributedText = self.subject.addAttributesTo(text: text)
         label.accessibilityIdentifier = AccessibilityIdentifiers.Shopping.ReviewQualityCard.subHeadlineLabel
         label.numberOfLines = 0
         label.adjustsFontForContentSizeCategory = true
         label.clipsToBounds = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
```

Comment:
This label should have a font.

Reply:
the font is set by the `MarkupAttributeUtility` obj which is instantiated with the base font, explicitly setting it in the label would be reduntant for the labels that have bold strings

Reply:
In that case the assignment of the `subHeadlineLabel` should be removed a few lines above. As it currently is implemented it gets a text here and an attributed text in configure.